.gitignore

```diff
@@ -1,8 +1,9 @@
+# Chart dependencies
+/charts/*/charts
 .DS_Store
 infra-*/kubeconfig
 infra-*/terraform.tfstate
 infra-*/terraform.tfstate*
 infra-*/.terraform*
 infra-*/secrets.auto.tfvars
 my-notes
-values.yaml
```

Comment:
Why did you remove `values.yaml` ? Isn't it still helpful to be able to install this the "old" way using git for development purposes?

Reply:
Since this is a repo containing a Helm chart (maybe more than 1 in the future) with a `values.yaml` file, we should not ignore it. Users will still be able to install the chart from source with no issues.